guides/Other Subjects/PieceStores.md

```diff
@@ -158,38 +86,56 @@ This is great and all, but we need to register these pieces/store:
 
 ```javascript
 const { Client } = require('klasa');
-const GenreStore = require('./GenreStore');
+const RawEventStore = require('./RawEventStore');
 
-class MySwankyMusicBot extends Client {
+class MyBot extends Client {
 
 	constructor(...args) {
 		super(...args);
 
-		// Make a new GenreStore and attach it to client
-		this.genres = new GenreStore();
+		// Make a new RawEventStore and attach it to client
+		this.rawEvents = new RawEventStore();
 
-		// Register the GenreStore to be loaded, init, and available
+		// Register the RawEventStore to be loaded, init, and available
 		// to be used as an arg to be looked up in commands
-		this.registerStore(this.genres);
-
-		// Note: You will likely want to make a custom Argument for this new piece type
+		this.registerStore(this.rawEvents);
 	}
 
 }
 
-new MySwankyMusicBot().login('token-goes-here');
+new MyBot().login('token-goes-here');
 ```
 
-Then we use it in our player class I completely made up earlier like so:
+Now to actually make our raw events work! This code goes in events/raw.
```

Comment:
`Now, to make our raw event store actually work, we're going to add this code in events/raw to run on every raw event received`